docs/src/pages/landing-page/Index.vue

```diff
@@ -298,4 +311,9 @@ q {
     }
   }
 }
+
+:deep(.q-btn--outline:before) {
```

Comment:
Always document these kind of rules, explaining why you used a `:deep` selector (since it's usually better to avoid them)